torch/csrc/jit/backends/backend_detail.h

```diff
@@ -58,11 +58,16 @@ TORCH_API void registerBackendPreprocessFunction(
     const std::string& name,
     const BackendPreprocessFunction& preprocess);
 
-TORCH_API bool hasBackendPreprocessFunction(const std::string& name);
+bool hasBackendPreprocessFunction(const std::string& name);
 
-TORCH_API BackendPreprocessFunction
+BackendPreprocessFunction
```

Comment:
I think you can move this to the same line (otherwise CLANFORMAT will likely complain).